docs/api-reference/legacy.rst

```diff
@@ -120,7 +120,7 @@ The API endpoint served at `upload.pypi.org/legacy/
 <https://upload.pypi.org/legacy/>`_ is Warehouse's emulation of the
 legacy PyPI upload API. This is the endpoint that tools such as `twine
 <https://twine.readthedocs.io/>`_ and `distutils
-<https://docs.python.org/3.7/distutils/packageindex.html#the-upload-command>`_
+<https://docs.python.org/3.8/distutils/packageindex.html#the-upload-command>`_
```

Comment:
This link hasn't existed since 3.6, we should probably just talk about `twine` here.